posts/business-leaders-and-digital-transformation.md

```diff
@@ -0,0 +1,59 @@
+
+Now that we know the problem, how do we solve it? By Integrating decision making across technical and business leadership through effective communication. 
+
+There are five stages of software development that should be communicated throughout the software development cycle. Setup decisions, Product-management practices, Product-development practices, Product-delivery practices, and Enablers. Below, we will describe what each stage means and why it's important. 
+
+## Setup decisions
+1. When developing a new product, or extending an existing solution, business leaders have a vested interest in the tools and architecture used. The benefits of communicating the technical advantages of each approach early on are:
+    * Allowing for business stakeholders to negotiate existing partnerships or explore new ones with technology providers to create strategic business advantages that would further the success of the initiative.
+
+## Product-management practices
+1. In order to communicate business value behind software development initiatives, effective product management practices must be deployed. This includes communication to business executives throughout the product development lifecycle, a process that consists of several steps from numerous teams:
+    * Storyboards, customer feedback, market analysis, success metrics or benchmarks (such as KPIs or OKRs), and team performance updates.
+    * Leadership needs to have a firm understanding of what works and what doesn't. By creating that transparency, executives can support the management styles that are most effective at lower levels of the organization, allow the teams to pivot and iterate quickly, leverage team member's skill sets, enable the team to become thought leaders and ultimately enrich management at every level.
+
+## Product-development practices
+1. Metrics around release schedules, testing and the extensibility of application interfaces allow business leaders to get insights into the entire lifecycle of a software development initiative. These include:
+    * Schedules allow them to have a baseline to measure progress against.
+    * Testing provides insight into how the development is progressing.
+    * Application interfaces allow them to think ahead in terms of strategy and growth. 
+
+## Product-delivery practices
+1. In cloud-based development, the use of tools to gather data-driven insights, polling customer engagement through the use of selective feature releases, and leveraging open-source tools to reduce development time and the cost of maintenance. 
```

Comment:
Does this thought need to be concluded?  Seems as though it's missing something at the end.

Reply:
passed this feedback to ibrahim and he finished the sentence properly. changes have been pushed